xwiki-platform-core/xwiki-platform-web/src/main/webapp/templates/accessdenied.vm

```diff
@@ -6,21 +6,25 @@
   $response.setStatus(403)
   ## Otherwise, unauthenticated user with Basic authentication, a 401 status has been set in Java.
 #end
-#if("$!request.ajax" != '')
+#if ("$!request.ajax" != '')
 $services.localization.render('notallowed')
 #else
 #template("startpage.vm")
 <div class="main layoutsubsection">
 <div id="mainContentArea">
-#if ($xwiki.isReadOnly())
-#xwikimessageboxstart($services.localization.render('error') $services.localization.render('readonly'))
+#if ($xwiki.exists("XWiki.AccessDenied"))
+  $xwiki.includeForm("XWiki.AccessDenied",false)
```

Comment:
you're going to find that I nitpick (and you'll be right ;)), but could you add a space after `,` as in `, false`? ;)

Reply:
let me add a space indeed, I was about to do it in the first place and was hesitating